### PR TITLE
Add confirm flag to `mc admin update --yes`

### PR DESCRIPTION
**Pull request by dvaldivia**
Adds a `--yes` or `-y` flag to `mc admin update` to bypass the new upgrade confirmation prompt

Signed-off-by: Daniel Valdivia <hola@danielvaldivia.com>

### CODE REVIEW

File: cmd/admin-update.go [hunk 1]
```diff
@@ -30,13 +30,20 @@ import (
 	"github.com/minio/pkg/console"
 )
 
+var adminUpdateFlags = append([]cli.Flag{
+	cli.BoolFlag{
+		Name:  "yes, y",
+		Usage: "Confirms the server update",
+	},
+}, subnetCommonFlags...)
```
Comment (vadmeste) [inline cmd/admin-update.go:38]: not sure why this is needed subnetCommonFlags, can you explain ?

File: cmd/admin-update.go [hunk 2]
```diff
@@ -106,7 +113,9 @@ func mainAdminServerUpdate(ctx *cli.Context) error {
 
 	updateURL := args.Get(1)
 
-	if isTerminal() {
+	autoConfirm := ctx.Bool("yes")
+
+	if isTerminal() && !autoConfirm {
```
Comment (vadmeste) [inline cmd/admin-update.go:118]: No issue with this but I just wanted to understand how isTerminal() is not enough, in which environment is this needed ?

**Review by vadmeste — COMMENTED** [review]
Few comments

File: cmd/admin-update.go (hunk 1 above)
Comment (dvaldivia) [reply cmd/admin-update.go:38]: that's a mistake on my part, fixed

File: cmd/admin-update.go (hunk 2 above)
Comment (dvaldivia) [reply cmd/admin-update.go:118]: Not sure how the prompt could be bypassed due to `isTerminal()` by adding the explicit flag and showing it on help users can discover this

**Comment by pjuarezd** [comment]
Tested 
```sh
./mc admin info minio-tenant-1 --insecure 
●  minio-tenant-1-ss-0-0.minio-tenant-1-hl.minio-tenant-1.svc.cluster.local:9000
   Uptime: 1 minute 
   Version: 2023-01-02T09:40:09Z
   Network: 3/3 OK 
   Drives: 2/2 OK 
   Pool: 1

●  minio-tenant-1-ss-0-1.minio-tenant-1-hl.minio-tenant-1.svc.cluster.local:9000
   Uptime: 1 minute 
   Version: 2023-01-02T09:40:09Z
   Network: 3/3 OK 
   Drives: 2/2 OK 
   Pool: 1

●  minio-tenant-1-ss-0-2.minio-tenant-1-hl.minio-tenant-1.svc.cluster.local:9000
   Uptime: 1 minute 
   Version: 2023-01-02T09:40:09Z
   Network: 3/3 OK 
   Drives: 2/2 OK 
   Pool: 1

Pools:
   1st, Erasure sets: 1, Drives per erasure set: 6

6 drives online, 0 drives offline

./mc admin update minio-tenant-1 --insecure --yes
Server `minio-tenant-1` updated successfully from 2023-01-02T09:40:09Z to 2023-01-06T18-11-18Z

./mc admin update minio-tenant-1 --insecure -y   
Server `minio-tenant-1` already running the most recent version 2023-01-06T18:11:18Z of MinIO
```

**Review by pjuarezd — APPROVED** [review]

**Review by cniackz — APPROVED** [review]
LGTM

**Comment by cniackz** [comment]
Please merge, is needed for automation 🙏

**Review by bexsoft — APPROVED** [review]

**Comment by cniackz** [comment]
Thank you @harshavardhana for the merge 🙏

**Pull request merged**